cfgov/scripts/validate_redirects.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+
+import requests
+from tablib import Dataset
+
+
+def load_redirects(redirects_filename):
+    csv = redirects_filename.endswith('.csv')
+    mode = 'rt' if csv else 'rb'
+
+    with open(redirects_filename, mode) as f:
+        dataset = Dataset().load(f, format='csv' if csv else None)
+
+    return [tuple(row[i].rstrip('*') for i in (5, 6)) for row in dataset]
```

Comment:
Could we maybe make 5 and 6 default values to arguments that are overwrite-able from the command line, ie `from_index=5, to_index=6` or some such? This seems to be keyed for a specifically formatted excel file but, if we need to rerun this in the future, it would be nice to not have to make this (albeit small) code change if the excel/csv format changes.